docs/rfc/0004_component_groups_scale_view.md

```diff
@@ -0,0 +1,299 @@
+|                    |                                |
+| -------            |--------------------------------|
+| Feature            | Scale View of Component Groups |
+| Status             | In-progress                    |
+| Date               | 2022-09-05                     |
+| Authors            | arkbriar                       |
+| RFC PR #           |                                |
+| Implementation PR #|                                |
+|                    |                                |
+
+# **Summary**
+
+The RFC aims to provide a new CR to support the scale subresource over groups of some user-specified component of
+the RisingWave resource. With that, the users can benefit from the scale subresource and apply any tools that have such
+requirement. For example, the horizontal auto-scaling service (HPA/[KEDA](https://github.com/kedacore/keda)) will work
+after this is implemented.
+
+# **Motivation**
+
+> Why are we doing this?
+
+The [scale subresource](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#scale-subresource)
+is
+a subresource like the status but exposes three fixed fields:
+
+- `.spec.replicas` which refers to the desired replicas
+- `.status.replicas` which refers to the current replicas
+- `.status.selector` which refers to the Pod selector (but in a serialized manner)
+
+However, the current design of the `RisingWave` resource can not provide such a subresource because it is consisted of
+four independent components, and each of the component can have one or more groups. Therefore, the `kubectl scale` can
+not work directly on the `RisingWave` resource and so do other tools depend on that. And it is the main reason for
+proposing this new CR. I named it 'RisingWaveComponentScaleView'.
+
+> What use cases does it support?
+
+With the `RisingWaveComponentScaleView`, the users can set up a view (similar to the view in databases) of some groups
```

Comment:
How about simplifying the CR name? all the scaling is for the components, maybe no necessary add the `component` in the CR name.
Just a little suggestion, after all, a long name is not convenient~.

Reply:
Good advice. Let me change it to `RisingWaveScaleView`.